src/components/apps/Notes/Notes.svelte

```diff
@@ -0,0 +1,145 @@
+<script>
+  import { theme } from '🍎/stores/theme.store';
+	import {addNote, notesStore} from './stores';
+	
+	const NEW_NOTE = {title: '', text: ''};
+	
+	let editing = false;
+	let note;
```

Comment:
Please define a type for this. The type could be defined as `type Node { id: string; /*Other properties */ }` on the top and be later referred to in this variables declaration as `let note: Note`.